prac_05/emails.py

```diff
@@ -0,0 +1,38 @@
+def main():
```

Comment:
Yea it so long man, maybe you could try like one function get the name from the dict and another checking the input and other error etc.
eg. 
def get_name(email):
    parts = email.split("@")[0].split(".")
    name = " ".join(parts).title()
    return name

Reply:
has been refactored to be simpler